be/src/common/config.h

```diff
@@ -763,6 +763,8 @@ CONF_Int32(quick_compaction_batch_size, "10");
 // do compaction min rowsets
 CONF_Int32(quick_compaction_min_rowsets, "10");
 
+CONF_mBool(enable_function_pushdown, "true");
```

Comment:
Not enable it by default, the new feature should be an experimental feature and open it by default when 1.3 is released.

Reply:
ok done